release-notes/release-notes59.md

```diff
@@ -131,7 +131,15 @@ For information on the Table plugin, see: [Table plugin]({{site.baseurl}}/plugin
 
 The following enhancements were made for the {{site.productname}} 5.9 release.
 
-### Enhancement name
+## Context Toolbars
```

Comment:
I wanted to avoid mentioning `positioning` in the heading as this also includes the transitioning. Maybe something like this instead?
```suggestion
### Improved Context Toolbars user experience
```